src/go/agwd/config/config_test.go

```diff
@@ -258,7 +258,7 @@ func TestNewConfigManager_DefaultNotFound(t *testing.T) {
 
 	cm := NewConfigManager()
 	err := LoadConfigFile(cm, filepath.Join("testdata", "doesnotexist.json"))
-	assert.True(t, os.IsNotExist(errors.Cause(err)))
+	assert.True(t, os.IsNotExist(errors.Unwrap(err)))
```

Comment:
This is not discussed in https://github.com/magma/magma/issues/12632, can you add it?

Reply:
I added a bullet point to the list in my comment there.